### DISCUSSION TITLE
Any way to extract html tag's contents from `templateContent` ?

### DISCUSSION THREAD

**Discussion by amithm7** [11ty/eleventy#1747] [Q&A]
#1266 maybe related.

I have a set of markdown files, which I am using to display elsewhere with collections feature.

using nunjucks for loop: `{{ item.templateContent | safe }}` would parse the whole markdown as it is.

But I would like to use it as html `<details>` tag to display it. Any way to parse content of html / md?

**Comment by pdehaan** [comment]
For HTML parsing, I generally use [**cheerio**](https://www.npmjs.com/package/cheerio) (although I've never used it w/ 11ty; but theoretically it should be possible to do some HTML parsing using a custom filter, if needed).
